### DISCUSSION TITLE
the poor fitting of the elimination phase

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1257] [Questions & Problems]
Hello everyone, I would like to ask about the possible causes of the poor fitting of the elimination phase.
![消除相1](https://user-images.githubusercontent.com/95362728/180151718-1a9b09ea-82fa-48db-8f32-e9316ac3d96d.jpg)
![2](https://user-images.githubusercontent.com/95362728/180151741-f7a8994e-c206-416a-b607-c632593dd247.jpg)

**Comment by StephanSchaller** [comment]
This could have many reasons and is impossible to solve here, if no info is provided on the compound (PhysChem Properties, is it charged? what distribution method was used, etc, etc....). 
Or you just attach the model file as a *.zip file here...

**Reply by WTY-tt** [reply]
Thank you very much！
[利伐沙班.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/9247045/default.zip)

**Comment by Yuri05** [comment]
You have added some enzymes/transporters to you individuals, but their initial concentration in some individuals is zero everywhere
E.g. in the individual **20mg-12例-64.7** which is used in the simulation above all enzymes and transporters are zero everywhere and thus have no effect on the simulation result. Probably not what you want...
![grafik](https://user-images.githubusercontent.com/25061876/182596399-9463a5d6-035a-41e8-8bd7-45236e49b5dd.png)

**Reply by WTY-tt** [reply]
Thank you very much for your answer, which is very helpful. I neglected this part, and would like to ask about how to obtain the initial concentration here.

**Reply by WTY-tt** [reply]
Thank you very much for your answer. I found that the initial concentration of enzyme/transporter in different tissues was not selected when I chose enzyme or transporter.

**Reply by Yuri05** [reply]
If you don't have any knowledge about initial concentrations of interest, the first approach would be to load the expression profile from the expression database
* in PK-Sim 10: https://docs.open-systems-pharmacology.org/v/v10/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-data#adding-protein-quantities-by-querying-the-internal-database 
* in PK-Sim 11: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile#editing-protein-expression-by-querying-the-expression-database

After that it might be required to fit the **Reference Concentration** of Enzyme/Transporter: either manually or via the Parameter IDentification Tool

**Reply by WTY-tt** [reply]
Thank you very much for your answer. I have imported it from the database.

**Comment by regulus92** [comment]
If you use more observed PK data for validation, and taken together, this fitting is probably good and acceptable. Have a check on prediected/observed Cmax and AUC. My point of view.
